tests/test_sim_client.py

```diff
@@ -180,7 +194,9 @@ def determine_expected_state(self, topic):
             var = variables[variable]
             if var["topic_type"] == "READ":
                 if var["idl_type"] == "boolean":
-                    expected_state[variable] = False
+                    expected_state[variable] = True
+                    if "ALARM" in variable:
+                        expected_state[variable] = False
```

Comment:
Either I am completely confused or your doc string for this method needs an update. From reading this snippet of code it looks like you expect booleans to be True unless they have an alarm state, but the doc string says `The expected values either is set to False (in case of a boolean)...` which suggests that this should always be False.

Please update the doc string, if it is outdated, or add some documentation explaining what's going on.

Also a gentle suggestion to remove the `s` from `values` in `The expected values either  is set to...`

Reply:
I updated the doc string since it indeed was outdated.